panoptes_client/subject_set.py

```diff
@@ -103,6 +104,21 @@ def remove(self, subjects):
 
         return self.links.subjects.remove(subjects)
 
+    def subject_statuses(self, workflow_id):
+        """
+        A generator which yields :py:class:`.SubjectWorkflowStatus` objects for subjects in this
+        subject set.
+
+        Examples::
+
+            for status in subject_set.subject_statuses(1234):
+                print(status.retirement_reason)
+        """
+
+        subject_ids = ', '.join((subject.id for subject in self.subjects))
+        for status in SubjectWorkflowStatus.where(subject_id=subject_ids, workflow_id=workflow_id):
```

Comment:
OOI - is this meant to use the `subject_ids` param?

Reply:
I might not have understood the question, but if you are talking about ``SubjectWorkflowStatus.where(subject_id=subject_ids, workflow_id=workflow_id)``  will also produce the same result as `SubjectWorkflowStatus.where(subject_ids=subject_ids, workflow_id=workflow_id)`. So I can update to the latter for better readability